posts.md

```diff
@@ -0,0 +1,16 @@
+---
+title: All Posts
+permalink: /posts/
+layout: page
+excerpt: All posts I've made.
+comments: false
+---
+
+{% for post in site.posts%}
+<article class="post-item">
+  <span class="post-item-date">{{ post.date | date: "%b %d, %Y" }}</span>
+  <h3 class="post-item-title">
```

Comment:
Double border issue, To remove change h3 element to h4 element 
![image](https://user-images.githubusercontent.com/20749264/88260446-95755980-ccee-11ea-9b4c-a057c13f371e.png)